app/views/users/show.html.erb

```diff
@@ -1 +1,34 @@
-<h1>This is the show user section for testing</h1>
\ No newline at end of file
+ <div class="container-sm" style="Margin-top: 20px">
+   <!-- app/views/posts/index.html.erb -->
+   <%= render partial: 'shared/user', locals: { user: @user } %>
+  </div>
+  <div class="card border-success mb-3">
+    <div class="card-body text-success text-bg-light p-3">
+      <h5 class="card-title">Bio</h5>
+      <p class="card-text"><%= @user.bio%></p>
+    </div>
+  </div>
+  <% @recent_posts.each do |post|%>
+    <div class="card border-success mb-3">
+      <div class="card-body text-success">
+        <a href = <%= user_post_path(post.author.id, post.id) %> style="text-decoration: none;">
+          <h5 class="card-title">Post <%= post.title %></h5>
+        </a>
+        <p class="card-text">
+          <%= post.text %>
+        </p>
+        <div class= "card-text">
+          <span class="card-text">Comments: <%= post.comment_counter%></span>
+          <span class="card-text">Likes: <%= post.likes_counter%></span>
+        </div>
+      </div>
+    </div>
+  <% end %>
+  <%= will_paginate @recent_posts %>
+  <div class="d-grid gap-2 col-6 mx-auto">
+    <a href = '<%= user_posts_path(@user) %>'>
+      <button type="button" class="btn btn-outline-primary" disabled>see all posts</button>
+    </div>
+  </div>
```

Comment:
[OPTIONAL BUT RECOMMENDED] Well done :clap:  you did a nice job but the previous reviewer :point_up:  didn't mention on which page you will implement this pagination functionality. You will implement it in the `/users/:user_id/posts` page but you implemented it on the `/users/:user_id` page which is not correct if you see the wireframe, you will see this route `/users/745/posts` where the pagination is implemented so kindly fix this issue and please don't skip this requirement this is optional because the previous reviewer missed that requirement and also please display only most recent posts in `users/:user_id` page

![Screenshot from 2024-01-27 23-27-04](https://github.com/vikas924/rails-blog/assets/45226848/9f22d7fb-5e1f-4223-9775-1018939485e3)

Reply:
@aamirkhan2478 can you review my pull request, I don,t have much time left for submission. I have one pull request open